database/factories/PositionFactory.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/** @var \Illuminate\Database\Eloquent\Factory $factory */
+
+use App\Position;
+use Faker\Generator as Faker;
+
+$factory->define(Position::class, function (Faker $faker) {
+    return [
+        'callsign' => strtoupper($faker->word).'_'.$faker->randomElement(['TWR', 'GND', 'DEL', 'APP', 'ATIS', 'CTR']),
```

Comment:
`strtoupper($faker->lexify('????'));` would likely be more suitable for generating an ICAO.

Reply:
Agreed!